### PR TITLE
home page

### PR DESCRIPTION
**Pull request by dandelionadia**

### CODE REVIEW

File: src/atoms/Button.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Button() {
+    return (
+        <div>
```
Comment (kettanaito) [inline src/atoms/Button.js]: Don't wrap button in a `div`. Remove the `div`.

File: src/atoms/Avatar.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Avatar() {
+    return (
+        <div className="box">
```
Comment (kettanaito) [inline src/atoms/Avatar.js]: Maybe you don't need to have an extra `div` around. Try removing it and changing the CSS.

File: src/atoms/Button.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Button() {
+    return (
+        <button className="button">Button Lor </button>
```
Comment (kettanaito) [inline src/atoms/Button.js]: This always renders a button with the text "Button Lor". Instead, allow to use any text:

```jsx
function Button(props) {
  return <button className="button">{props.children}</button>
}
```

Now you can render your `Button` component with different text anywhere:

```jsx
<Button>Foo</Button>
<Button>This is another text</Button>
```

File: src/atoms/logo.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
+.header__logo {
+    display: flex;
+    align-items: center;
+}
+
+.logo {
+width: 100px;
+height: 60px;
```
Comment (kettanaito) [inline src/atoms/logo.scss]: Indentation?

File: src/components/HomeInfo.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    render() {
+        return (
+            <div className="container-homeInfo">
+                <h3>subtitle</h3>
+                <h1>title to tilorem</h1>
+                <div className="container-messager">
+                    <Avatar />
+                    <span className="message">"It is a long established fact that a reader will be distracted by the readable content of a page."</span>
+                </div>
+                <Button />
```
Comment (kettanaito) [inline src/components/HomeInfo.js]: After you fix the Button component, you can pass here its children to be its text:

```jsx
<Button>Some text you want</Button>
```

File: src/components/Menu.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    render() {
+        const { menu } = this.state
+
+        return (
+            <div className="menu">
+                <ul className="menu__list">
+                    {menu.map((item) => (
+                        <li className="menu__item">
```
Comment (kettanaito) [inline src/components/Menu.js]: Each `li` should have a unique `key` prop.

File: src/pages/HomePage.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+import { Logo } from '../atoms/Logo';
+
+function HomePage() {
+    return (
+        <div className="container-homePage">
+            <div className="header">
+                <Logo />
+                <Menu />
+            </div>
+            <div className="container-infoPage">
```
Comment (kettanaito) [inline src/pages/HomePage.js]: This is not a good class name :P

File: src/style/main.scss [hunk 8]
```diff
@@ -0,0 +1,7 @@
+@import './variables.scss';
+@import '../pages/homePage.scss';
+@import '../components/menu.scss';
+@import '../atoms/logo.scss';
+@import '../components/homeinfo.scss';
+@import '../atoms/button.scss';
+@import '../atoms/avatar.scss';
```
Comment (kettanaito) [inline src/style/main.scss]: `.scss` extension is not required. You can leave it out.

File: src/atoms/Avatar.js (hunk 2 above)
Comment (dandelionadia) [reply src/atoms/Avatar.js]: I need this box becose it makes `overflow: hidden;`

File: src/atoms/logo.scss (hunk 4 above)
Comment (dandelionadia) [reply src/atoms/logo.scss]: yes)

File: src/atoms/Avatar.js (hunk 2 above)
Comment (kettanaito) [reply src/atoms/Avatar.js]: Got it!

File: src/components/HomeInfo.js [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+    render() {
+        return (
+            <div className="container-homeInfo">
+                <h3>subtitle</h3>
+                <h1>title to tilorem</h1>
+                <div className="container-messager">
+                    <Avatar />
+                    <span className="message">{this.props.children}</span>
+                </div>
+                <Button children={'Button Lor'} />
```
Comment (kettanaito) [inline src/components/HomeInfo.js]: You don't have to give `children` as an attribute. It's a _special_ prop that which value is anything you give _inside_ the component:

```diff
- <Button children="Button Lor" />
+ <Button>Button Lor</Button>
```

These two are exactly the same. Try to use the second one.

File: src/components/Menu.js [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    render() {
+        const { menu } = this.state
+
+        return (
+            <div className="menu">
+                <ul className="menu__list">
+                    {menu.map((item) => (
+                        <li key={item.toString()} className="menu__item">
```
Comment (kettanaito) [inline src/components/Menu.js]: You can also use `index` if your items will not change (won't be deleted or re-ordered):

```jsx
{menu.map((item, index) => (
  <li key={index}>...</li>
)}
```

File: src/pages/HomePage.js [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+
+function HomePage() {
+    return (
+        <div className="container-homePage">
+            <div className="header">
+                <Logo />
+                <Menu />
+            </div>
+            <div className="infoPage">
+                <HomeInfo children={"It is a long established fact that a reader will be distracted by the readable content of a page."} />
```
Comment (kettanaito) [inline src/pages/HomePage.js]: Give the text _inside_ the component:

```jsx
<HomeInfo>Your text here</HomeInfo<
```

File: src/components/HomeInfo.js [hunk 12]
```diff
@@ -12,7 +12,7 @@ class HomeInfo extends React.Component {
                     <Avatar />
                     <span className="message">{this.props.children}</span>
                 </div>
-                <Button children={'Button Lor'} />
+                <Button>Button Lor</Button>
```
Comment (kettanaito) [inline src/components/HomeInfo.js]: This looks perfect.

**Pull request merged**